LocalSettings.php

```diff
@@ -560,14 +560,14 @@
 			'Use Public Test Wiki' => 'Please use Public Test Wiki, https://publictestwiki.com, to test the administrator and bureaucrat tools (as well as Miraheze since the wiki is hosted by us). You should review and follow all TestWiki:Policies, especially TestWiki:Testing policy and TestWiki:Main policy, reverting all tests you perform in the reverse order which you performed them. Request permissions at TestWiki:Request permissions. Thank you.',	
 			'Database exists (wiki still active)' => 'Wiki database name and subdomain already exist. Please visit the local wiki and contribute there. Please reach out to any local bureaucrat to request any permissions if you require them. If bureaucrats are not active on the wiki after a reasonable period of time, please start a local election and ask a Steward to evaluate it on the Stewards\' Noticeboard. Thanks.',
 			'Database exists and the requester used to be a bureaucrat (wiki already deleted)' => 'Wiki database name already exists, but the wiki itself has already been deleted in accordance with the Dormancy Policy. I will request a Steward to undelete it for you. When it has been undeleted and reopened, please visit the local wiki and ensure you make at least one edit or log action every 45 days. Wikis are only deleted after 6 months of complete inactivity; if you require a Dormancy Policy exemption, you should review the policy and request it once your wiki has at least 40-60 content pages. Thank you.',
 			'Duplicate request' => 'Declining as a duplicate request, which needs more information. Please do not edit this request and instead go back into your original request. Also, please do not submit duplicate requests. Thank you.',
 			'Content Policy (unsubstantiated insult)' => 'Declining per Content Policy provision, "Miraheze does not host wikis with the sole purpose to spread unsubstantiated insult, hate or rumours against a person or group of people." Thank you for understanding.',
 			'Content Policy (makes it difficult for other wikis)' => 'Declining per Content Policy provision, "A wiki must not create problems which make it difficult for other wikis." Thank you for understanding.',	
-			'Duplicate wiki' => 'We already have a wiki that covers this topic. Please contribute to that wiki instead. Thank you.',
+			'Duplicate wiki' => 'Your proposed wiki appears to duplicate, either substantially or entirely, the content of an existing wiki (see the "request comments" tab for one or more link(s) to the existing wiki(s)). Can you please describe in a few more sentences by adding to, but not replacing, your existing description, the scope and focus for your wiki, and also assure us that your wiki will not be a complete or substantial duplication? Thank you.',
```

Comment:
Although this is minor, could you please say "could" instead of "can" and "Request Comments tab" instead of "request comments tab"?

Reply:
> Although this is minor, could you please say "could" instead of "can" and "Request Comments tab" instead of "request comments tab"?

@R4356th Okay, sure. I'm never sure whether to use _could_ or _can_, so not fussed about that. Not fussed about the other, really, but _do_ mildly prefer lowercase. Not a big deal, though, I will change it. I don't feel _tab_ should be enclosed in the "Request Comments" quotation marks as the title of the tab is _Request Comments_. So I'll make that minor amendment as well.

Reply:
@R4356th Should be done now. We could also remove the quotation marks around _Request Comments_, if you prefer.

Reply:
Thank you.